charts/grafana/templates/_pod.tpl

```diff
@@ -857,6 +857,9 @@ containers:
     volumeMounts:
       - name: sc-plugins-volume
         mountPath: "/etc/grafana/provisioning/plugins"
+      {{- with .Values.sidecar.plugins.extraMounts }}
+      {{- toYaml . | trim | nindent 6 }}
```

Comment:
why trim?

Reply:
not really required, I just left it the same as other extraMounts (like the sc-alerts one)